packages/reactive-dict/reactive-dict.js

```diff
@@ -164,6 +164,23 @@ _.extend(ReactiveDict.prototype, {
     
   },
 
+  delete: function(key) {
+    var self      = this;
+    var oldKeys   = self.keys;
+    var didRemove = false;
+
+    if (_.has(oldKeys, key)) {
+      changed(self.keyDeps[key]);
```

Comment:
Sorry, I didn't mean API-wise, I meant to simplify the implementation and reduce duplication of code.

This change would also have a side effect that code like `Session.get("asdf")` would no longer be invalidated when `asdf` goes from `undefined` to absent (in which case the code still returns `undefined`), but that seems like an improvement if anything.